views/js/controller/tests/editTest.js

```diff
@@ -22,16 +22,26 @@
  */
 define([
     'jquery',
-    'ui/lock'
-],	function($, lock){
+    'ui/lock',
+    'module',
+    'layout/actions',
+],	function($, lock, module, actions){
     'use strict';
 
     return {
 
+
         /**
          * Controller's entrypoint
          */
         start(){
+
+            const config = module.config();
+            const previewAction = actions.getBy('test-preview')
```

Comment:
Missing semi-colon

Reply:
You can use some multi-declarating way like 

```
 const config = module.config(),
            previewAction = actions.getBy('test-preview');
```

If you want, but for me, only semi-colon required for don't break our FE rules